src/config/env.ts

```diff
@@ -14,6 +14,8 @@ export const IN_PRODUCTION =
   process.env.NODE_ENV === 'production' &&
   process.env.REACT_APP_VERCEL_ENV !== 'preview';
 
+export const IN_DEVELOPMENT = process.env.NODE_ENV === 'development';
```

Comment:
I could use !IN_PRODUCTION, but it feels safer to be explicit about "in development" than not prod.